release-notes.md

```diff
@@ -1,3 +1,34 @@
+2.0.0-RC7
+---------
+* CI/CD implementation
+* Delivery Tracker - limit requests map per peer
+* add multiple fields to the /connectedPeer endpoint response
```

Comment:
@paologalligit pls confirm, I think we don't have this, it is going to be added directly to the SDK, right?

Reply:
Correct, this is just an sdk change

Reply:
so this one won't be merged now?
https://github.com/HorizenOfficial/Sparkz/pull/8

Reply:
No, we're going to close it once the sdk-97 on Sidechains-SDK will be merged

Reply:
Ok, removed that line, thanks